### ISSUE TITLE
Feature request: "just --list": allow ordering by order in justfile 

### ISSUE THREAD

**Issue by dionjwa** [casey/just#667]
The default help `just --list` is great, and by default alphabetical ordering is often what you want.

But when I have a short list of very high level commands, sometimes I really want them to be e.g. 

```
Available recipes:
    build +args="" # Build the stack. 
    test           # Test the stack. 
    publish        # publish artifacts
```

In other words, I would like the order of the commands to contain extra information: the steps the user will typically need to follow to do some high level task.

Or, the order of the commands is how important they are, I don't want `just a-little-helper` to be above `just omg-deploy-our-entire-cloud-stack`. Those are contrived, but you get the idea. Ordering can help self-document.

I would propose to switch on file-level ordering by something like:

`just --list --disable-sort` or similar, so it keeps backwards compatibility.

Thanks for considering! And thanks for the fixes and updates, this is an excellent tool in my belt.

**Comment by casey** [comment]
Thanks for opening this issue, and thank you for the kind words!

I think this is totally reasonable. I added it in #669, with the long flag `--unsorted` and short flag `-u`. It works with `--list` and `-summary`.

What do you think about calling it `--unsorted`?

**Comment by casey** [comment]
Great! I merged #669, and just cut v0.7.2. The new version is available in Cargo, and prebuilt binaries should appear on the release page soon. Thanks for the feature request!

As a random aside, if Just ever gets a config file (like `~/.justrc`) then this could be a setting, if users prefer to always see items in source order.
